OrcTests/test_data/functional_valid/block.orc

```diff
@@ -0,0 +1,12 @@
+{- block.orc -- Orc program block
+ -
+ - Created by amp on Feb 6, 2015 10:08:44 PM
+ -}
+
+import site Block = "orc.lib.util.Block"
+
+Block() ; "Test"
+
+{- 
+OUTPUT:
+-}
```

Comment:
I'm not gonna do this because many of the orc files are also missing the trailing newline. We can add all the newlines at once later. Better not to add it to the merge.